www/front_src/src/Resources/Filter/Criterias/searchQueryLanguage/index.test.ts

```diff
@@ -57,10 +57,28 @@ const parsedSearch = [
     type: 'multi_select',
     value: [{ id: 0, name: 'Central' }],
   },
-  { name: 'search', object_type: null, type: 'text', value: 'h.name:centreon' },
+  {
+    name: 'host_categories',
+    object_type: 'host_categories',
+    type: 'multi_select',
+    value: [{ id: 0, name: 'Linux' }],
+  },
+  {
+    isOptimizedModeEnabled: true,
```

Comment:
This property shouldn't be here in my opinion. Instead, we should have a mapping somewhere indicating which criteria should be displayed in optimized mode and be queried whenever needed before the actual display